src/button.tsx

```diff
@@ -436,12 +436,13 @@ const Button: React.FC<
 };
 
 const useButtonLinkStyles = createUseStyles((theme) => {
-    const padding = 6;
+    const paddingY = 6;
+    const paddingX = 12;
     return {
         link: {
             display: 'inline-block',
             width: 'auto',
-            padding,
+            padding: `${paddingY}px ${paddingX}px`,
```

Comment:
wrong alignment in callouts and cards
https://ibit.ly/HwBP
https://ibit.ly/YsS-

And hover in this case is quite "ugly"
https://ibit.ly/JDv_

Reply:
Fixed. This problem was related to the buttongrouop component